pkgs/development/gnuradio-modules/mkDerivation.nix

```diff
@@ -7,8 +7,10 @@ mkDerivation:
 args:
 
 # Check if it's supposed to not get built for the current gnuradio version
-if (builtins.hasAttr "disabledForGRafter" args) &&
-(lib.versionAtLeast unwrapped.versionAttr.major args.disabledForGRafter) then
+if ((builtins.hasAttr "disabledForGRafter" args) &&
+(lib.versionAtLeast unwrapped.versionAttr.major args.disabledForGRafter)) ||
+((builtins.hasAttr "disabledForGRbefore" args) &&
+(lib.versionOlder unwrapped.versionAttr.major args.disabledForGRbefore)) then
```

Comment:
I wonder if it will be better if we use something like `gnuradioAtLeast` and `gnuradioOlder`, just like `python.pythonAtLeast` `python.pythonOlder`. But if that is the case, we are going to modify all the files under `pkgs/development/gnuradio-modules/`. I would say let's keep that for another PR

Reply:
> I wonder if it will be better if we use something like `gnuradioAtLeast` and `gnuradioOlder`, just like `python.pythonAtLeast` `python.pythonOlder`. But if that is the case, we are going to modify all the files under `pkgs/development/gnuradio-modules/`. I would say let's keep that for another PR

Sounds great to me! However, we might want to `throw` some message if `disabledForGRafter` is used, because that argument might be used outside of nixpkgs in other people's flakes or alike.

Reply:
For the record, I'm fine with the way things are now too.